pdf_report/load_pdf.py

```diff
@@ -0,0 +1,175 @@
+""" Loads the PDF summary report of RealTimeTrains data into an S3 bucket. """
+from os import environ, path
+import logging
+
+from email.mime.application import MIMEApplication
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from io import BytesIO
+from datetime import datetime
+
+from dotenv import load_dotenv
+from boto3 import client
+from botocore.exceptions import NoCredentialsError
+import botocore.exceptions
+from botocore.exceptions import ClientError
+from psycopg2 import connect, Error as psycopg2_error
+from psycopg2.extensions import connection, cursor
+from psycopg2.extras import RealDictCursor
+
```

Comment:
Is this supposed to be in the pull request?

Reply:
Thank you, good spot